File: framework/cei_golang/src/impl/restfulresponse.go
package impl

import (
	"fmt"
	"io/ioutil"
	"net/http"
)

type RestfulResponse struct {
	response *http.Response
}

func newRestfulResponse(response *http.Response) *RestfulResponse {
	instance := new(RestfulResponse)
	instance.response = response
	return instance
}


func (inst *RestfulResponse) GetString() string {
	var body []byte
	defer inst.response.Body.Close()
	body, err := ioutil.ReadAll(inst.response.Body)
	if err != nil {
		fmt.Println(err)
		return ""
	}
	json := ParseFromString(string(body))
	return ""
}

File: framework/cei_java/src/main/java/cei/impl/RestfulResponse.java
package cei.impl;

import okhttp3.Response;

public class RestfulResponse {
    private final Response response;
    
    public RestfulResponse(Response response) {
        this.response = response;
    }
    
    public String getString() {
        String res = null;
        try {
            res = response.body().string();
            System.out.println(res);
        } catch (Exception e) {
            throw new CEIException("Cannot get string");
        }
        return res;
    }
}
